src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1475,8 +1475,10 @@ public void dragAndDropQuestionOption(String qnType, int qnNumber, int sourceInd
                 + targetIndex + "-" + qnNumber + "']//span[@class='glyphicon glyphicon-resize-vertical']"));
 
         Actions builder = new Actions(browser.driver);
-        // drag option 10 units above target and release
+        // drag option 10 units below target, move to target, move 10 units above target and release
```

Comment:
Can you provide some comments regarding why we do the extrat stuffs?

Reply:
@xpdavid Really sorry for such a late response. Been quite busy lately. I have added comments explaining the extra movements.

Actually I tried testing different combinations again this morning and realized that using 2 movements still works. Selenium 3 has added `moveInTicks` functionality [https://github.com/SeleniumHQ/selenium/blob/dbe5d01c93b54958c3955b36f5166a3f704b74c1/java/client/src/org/openqa/selenium/interactions/Actions.java#L413](https://github.com/SeleniumHQ/selenium/blob/dbe5d01c93b54958c3955b36f5166a3f704b74c1/java/client/src/org/openqa/selenium/interactions/Actions.java#L413) that adds a duration of 100 milisecond. It should hopefully solve this issue.